lib/src/main/java/statesAndCapitals/StatesAndCapitals.java

```diff
@@ -104,36 +104,36 @@ public void testStatesAndCapitals() throws IOException
         // Use findAny() or findFirst(), and orElseThrow()
         // Can use filter()
 
-        StateInfo cardinalState = null;
+        StateInfo cardinalState = states.stream().filter(s -> "cardinal".equals(s.getStateBird())).findAny().orElseThrow();
 
         testResults.put("I1", StatesAndCapitalsCheck.int1(cardinalState));
 
         // I2. Find if any state's lowest elevation is less than 0
         // Use anyMatch()
 
-        Boolean isAnyStateLessThan0Elevation = null;
+        Boolean isAnyStateLessThan0Elevation = states.stream().anyMatch(s -> s.getLowestElevation() < 0);
```

Comment:
Make sure the name of the method here exactly matches the getter in the StateInfo class (see also lines 121 and 136)